### DISCUSSION TITLE
Finding the underlying filesystem locations for a mergerfs file/dir?

### DISCUSSION THREAD

**Discussion by ar-jan** [trapexit/mergerfs-tools#128] [General]
I went through the merferfs-tools and and mergerfs readme but didn't find this (maybe I've missed it). Is there a command one can issue to find the underlying filesystem location(s) for a mergerfs file/dir? This https://github.com/trapexit/mergerfs#file--directory-xattrs looks relevant but I'm not sure how to use it.

I'm thinking of something like `mergerfs.locate /srv/mergerfs/mypool/path/to/file.txt` which would then output something like `/mnt/disk3/mergerfspool/path/to/file.txt`.

**Comment by trapexit** [comment]
Moved to discussions.

You use the generic xattr tooling as shown in the docs

```
getfattr -n user.mergerfs.allpaths /file
```

**Reply by ar-jan** [reply]
Thanks

**Comment by brandonleon** [comment]
I was thinking the same thing, feel free to use my iteration: https://codeberg.org/brandonleon/mergerfs-tools/src/branch/master/src/mergerfs.locate
